packages/client/src/lib/interface.ts

```diff
@@ -13,10 +13,12 @@ export type Tagged<T, Tag> = T & { tag?: Tag }
 export interface Service {
   endpoint: URL
   token: string
+  version?: string
```

Comment:
Please can we just use endpoint?

Reply:
https://github.com/ipfs-shipyard/nft.storage/pull/436